cirq-core/cirq/experiments/readout_confusion_matrix.py

```diff
@@ -178,7 +178,7 @@ def _confusion_matrix(self, qubits: Sequence['cirq.Qid']) -> np.ndarray:
             ein_input.extend([cm.reshape((2, 2) * len(qs)), self._get_vars(qs)])
         ein_out = self._get_vars(qubits)
 
-        # TODO(#5757): remote type ignore when numpy has proper override signature.
+        # TODO(#5757): remove type ignore when numpy has proper override signature.
```

Comment:
note that with https://github.com/quantumlib/Cirq/pull/5781 having been merged, mypy will now warn if it encounters `type: ignore` comments that are not needed, so we can probably just remove (or "remote" :-) ) these comments.